src/generateAppendClassName.js

```diff
@@ -1,4 +1,4 @@
-import Map from 'es6-map';
+import Map from './simple-map';
```

Comment:
I'm not sure that is fitting, as keys are not weakly referenced like in a WeakMap. Your call though.